episodes/filetypes-and-metadata.md

```diff
@@ -0,0 +1,498 @@
+Napari viewer.
+
+How do we set the pixel size in Napari? Most of the time, if the pixel size is 
+provided in the image metadata, napari-aicsimageio will set it automatically. 
+We can check this by running the following in Napari's console:
+
+```python
+# Get the first image layer
+image_layer = viewer.layers[0]
+
+# Print its scale
+print(image_layer.scale)
+```
+
+```output
+# [z y x]
+[0.35 0.2047619 0.2047619]
+```
+
+Each image layer in Napari has a `.scale` which is equivalent to the pixel size. 
+Here we can see that it is already set to values matching the image metadata. 
+
+If the pixel size isn't listed in the metadata, or napari-aicsimagio doesn't 
+read it correctly, you can set it manually like so:
+```python
+image_layer.scale = (0.35, 0.2047619, 0.2047619)
+```
+
+::::::::::::::::::::::::::::::::::::: challenge 
+
+## Pixel size / scale
+
+Copy and paste the following into Napari's console to get `image_layer_1`, 
+`image_layer_2` and `image_layer_3` of the yeast image:
+
+```python
+image_layer_1 = viewer.layers[0]
+image_layer_2 = viewer.layers[1]
+image_layer_3 = viewer.layers[2]
+```
+
+1. Check the `.scale` of each layer - are they the same?
+2. Set the scale of layer 3 to (0.35, 0.4, 0.4) - what changes in the viewer?
+3. Set the scale of layer 3 to (0.35, 0.4, 0.2047619) - what changes in the 
+viewer?
+4. Set the scale of all layers so they are half as wide and half as tall as 
+their original size in the viewer
+
+
+:::::::::::::::::::::::: solution 
+
+## Solution
+
+### 1
+
+All layers have the same scale
+
+```python
+print(image_layer_1.scale)
+print(image_layer_2.scale)
+print(image_layer_3.scale)
+```
+```output
+[0.35 0.2047619 0.2047619]
+[0.35 0.2047619 0.2047619]
+[0.35 0.2047619 0.2047619]
+```
+
+### 2
+```python
+image_layer_3.scale = (0.35, 0.4, 0.4)
+```
+You should see that layer 3 becomes about twice as wide and twice as tall as the 
+other layers. This is because we set the pixel size in y and x (which used to 
+be 0.2047619&mu;m) to about twice its original value (now 0.4&mu;m).
+
+### 3
+```python
+image_layer_3.scale = (0.35, 0.4, 0.2047619)
+```
+You should see that layer 3 appears squashed - with the same width as other 
+layers, but about twice the height. This is because we set the pixel size in y 
+(which used to be 0.2047619&mu;m) to about twice its original value (now 
+0.4&mu;m). Bear in mind that setting the pixel sizes inappropriately can lead 
+to stretched or squashed images like this!
+
+### 4
+
+We set the pixel size in y/x to half its original value of 0.2047619&mu;m:
+```python
+image_layer_1.scale = (0.35, 0.10238095, 0.10238095)
+image_layer_2.scale = (0.35, 0.10238095, 0.10238095)
+image_layer_3.scale = (0.35, 0.10238095, 0.10238095)
+```
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## Choosing a file format
+
+Now that we've seen an example of browsing metadata in Napari, let's look more 
```

Comment:
Did we also edit the metadata when we changed pixel size? Should it be:
"Now that we've seen an example of browsing and editing metadata" ?

Reply:
The pixel size is only changed within Napari (for display) - all the metadata remains unchanged.

Reply:
Thanks, let's leave as is then.